lib/alchemy/modules.rb

```diff
@@ -74,5 +87,20 @@ def action_matches?(navi, params)
     def remove_slash(str)
       str.gsub(/^\//, '')
     end
+
+    def self.validate_controllers_existence(controllers)
+      controllers.each do |controller_val|
+        next if controller_val.blank?
+
+        controller_name = "#{controller_val.camelize}Controller"
+
+        begin
+          controller_name.constantize
+        rescue RuntimeError
```

Comment:
Could we `rescue NameError` here instead? This is what Rails raises if the constant is undefined.
```suggestion
        rescue NameError
```